packages/patrol/example/ios/fastlane/Fastfile

```diff
@@ -0,0 +1,30 @@
+default_platform(:ios)
+
+git_authorization = "#{ENV["GIT_USERNAME"]}:#{ENV["GIT_TOKEN"]}"
+
+platform :ios do
+  desc "Signs app on CI"
+  lane :certificates do
+    setup_ci if ENV['CI']
+
+    app_store_connect_api_key(
+      key_id: ENV["APP_STORE_KEY_ID"],
+      issuer_id: ENV["APP_STORE_ISSUER_ID"],
+      key_content: ENV["APP_STORE_KEY"],
```

Comment:
Let's use `ENV.fetch()` here as well.

You might even want to lift these 3 variables to the top of this file, so it'll fail immediately when these aren't found, saving some debugging time.